gradle/check-licenses.gradle

```diff
@@ -30,17 +30,21 @@
 ext.acceptedLicenses = [
         'BSD License',
         'BSD 3-Clause',
+        'COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0',
         'Eclipse Public License - v 1.0',
         'Eclipse Public License - v 2.0',
         'Eclipse Public License 2.0',
         'Eclipse Public License 1.0',
+        'GPLv2+CE',
         'MIT License',
+        'The MIT License (MIT)',
         'Apache License, Version 2.0',
         'Bouncy Castle Licence',
         'Public Domain',
         'Mozilla Public License 1.0',
         'Mozilla Public License Version 1.1',
         'Mozilla Public License Version 2.0',
+        'No license found',
```

Comment:
Pretty sure this is not good to include

Reply:
Wondering if this is valid - the library in question is "public" licence - i.e. not licensed.

Reply:
Strange, I think they would want to at least limit liability, with the unlicensed https://choosealicense.com/licenses/unlicense/

Reply:
I'm pretty sure "No license found" means 100% copyright reserved and thus we can't touch it.  And in any case, we can't add 'No license found' to the accepted list because then any library that doesn't detect the license properly would slip through.

Reply:
Found a way to remove it - yay.